src/transports/auth_ntlm.c

```diff
@@ -5,8 +5,9 @@
  * a Linking Exception. For full terms see the included COPYING file.
  */
 
-#include "git2.h"
 #include "common.h"
+
+#include "git2.h"
```

Comment:
In fact, "common.h" should be dropped altogether. Instead, "auth_ntlm.h" should come first and "auth_ntlm.h" should have an include on "common.h" first